docs/Sample Queries/aws_ec2.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 1
+slug: '/ec2'
+---
+
+# AWS EC2
+
+## Create and update instances
+
+Install the AWS EC2 module
+
+```sql
+SELECT iasql_install('aws_ec2');
```

Comment:
Are we going to express the function calls as:

`SELECT iasql_install('aws_ec2');`
or
`SELECT * FROM iasql_install('aws_ec2');`

The second is longer but maybe the output is more SQLish?

Not related but maybe where we define the available functions we can make the comment that both formats are correct.

Reply:
Ah yes, it should be the second